src/helpers.rs

```diff
@@ -397,6 +397,26 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         Ok(())
     }
 
+    /// Helper function used inside the shims of foreign functions which reject the op
+    /// when isolation is enabled. It is used to print a warning/backtrace about the rejection.
+    fn report_rejected_op(&self, op_name: &str, reject_with: RejectOpWith) {
```

Comment:
This seems to be overlapping in terms of functionality with `isolation_error`.

Reply:
It's still used by ops which panic in isolation, as they're not converted to return an error.

Reply:
Okay. But we should somehow reflect this in the code. Maybe add a FIXME at `isolation_error` saying that this only exists for old ops not yet converted to return a proper error code?

Reply:
Sure, done.